org.openttd.OpenTTD.json

```diff
@@ -0,0 +1,205 @@
+{
+  "app-id": "org.openttd.OpenTTD",
+  "branch": "stable",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openttd",
+  "rename-icon": "openttd",
+  "rename-desktop-file": "openttd.desktop",
+  "finish-args": [
+    "--device=dri",
+    "--filesystem=home",
```

Comment:
Does it need home access?

Reply:
For exporting save files and additional music, afaik.